src/ui/atoms/canvas/Canvas.tsx

```diff
@@ -1,16 +1,11 @@
 /* eslint-disable max-lines-per-function */
 import React, { useEffect, useRef } from 'react'
 import { useAnimationFrame } from 'hook/useAnimationFrame'
-import type { DeepReadonly } from '../../../superTypes'
+import type { Callback, DeepReadonly } from 'superTypes'
 
-// eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
-export type TCanvasCreatedCallback = (canvas: HTMLCanvasElement) => void
-export type TCanvasDestroyedCallback = () => void
-export type TRenderCanvasCallback = (
-  // eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
-  canvas: HTMLCanvasElement,
-  deltaTime: number
-) => void
+export type TCanvasCreatedCallback = Callback<HTMLCanvasElement, void>
+export type TCanvasDestroyedCallback = Callback<void, void>
+export type TRenderCanvasCallback = Callback<{ canvas: HTMLCanvasElement; deltaTime: number }, void>
```

Comment:
Please, could remove the `T` prefix from types to be consistent with other ones?

Reply:
I can't in this PR. I've just re-writing the existing types on the Canvas component. Changing the naming would break the compatibility. I think it is wiser to refactor the types in a separate PR if you agree.

Reply:
@ccamel I fully agree 👌